src/notification/twilio.ts

```diff
@@ -0,0 +1,25 @@
+import {Config} from '../config';
+import {Logger} from '../logger';
+import twilio from 'twilio';
+
+const client = twilio(Config.notifications.twilio.sid, Config.notifications.twilio.auth);
+
+export function requestTwilioCall() {
+	let toNumber = Config.notifications.phone.number;
+
+	if (!toNumber.startsWith('+1')) {
+		toNumber = `+1${toNumber}`;
+	}
```

Comment:
Instead of having any of this logic, the user should be responsible for providing a correct country prefix, as well as their number. I do understand we are piggybacking off another config, but we should probably just update the docs to note this if it's required by Twilio.

Reply:
Good call on that. I do think that if we piggyback off the PHONE_NUMBER variable, there will be and problems with sms if we request the users to put the country code before the number to use Twilio since it's using email to send the text. I think there's two ways we can go with this:

1. Create a new config variable for the country code and add it to the PHONE_NUMBER programmatically (how it's working now, but with the new country code var).

2. Convert the sms functionality to use Twilio instead. That will get rid of any carrier checking, but the downside is that it will require the creation of a Twilio account. 

Let me know which route you'd prefer and I can implement either one.

Reply:
Definitely the former if anything. I'd be curious to do some testing with the email including the country code.

Reply:
Just did some testing with sprint and verizon and sending the text with the +1 before the number was a no go. I'll go ahead and push the changes with the new variable shortly.

Reply:
Thanks for testing!